lib/tasks/indices.rake

```diff
@@ -173,4 +173,31 @@ You should run this task if the index schema has changed.
       index_name: args.index_name
     )
   end
+
+  desc "Monitor the search indices and send data to statsd"
+  task :monitor_indices do
+    client = Services.elasticsearch
+    statsd = Services.statsd_client
+    missing = []
+
+    SearchConfig.instance.all_index_names.each do |index|
+      begin
+        stats = client.indices.stats index: index, docs: true
+        docs = stats["_all"]["primaries"]["docs"]
+
+        count = docs["count"]
+        statsd.gauge("#{index}_index.docs.count", count)
```

Comment:
Is there any value in just dumping all the index stats into statsd rather than extracting a couple of individual values?

I don't have any particular use in mind, but it might be useful for debugging in future.